README.md

```diff
@@ -1,3 +1,122 @@
-# vscode
+# VS Code
 
-The action runs VSCode in web browser.
+This action runs an instance of [VS Code](https://code.visualstudio.com/) and runs in in the default web browser.
+
+It requires the name of the image on which to run VS Code and references to the following 5 volumes: data, code, config, notebooks, and results. 
+These volumes will be mounted to /project/data, /project/modules, /project/config, /project/notebooks and /project/results respectively.
+
+### Quick example
+
+```
+jobs:
+  vscode:
+    action: gh:neuro-actions/vscode@master
+    args:
+      image: neuromation/base
+      volumes_data_remote: $[[ volumes.data.remote ]]
+      volumes_code_remote: $[[ volumes.code.remote ]]
+      volumes_config_remote: $[[ volumes.config.remote ]]
+      volumes_notebooks_remote: $[[ volumes.notebooks.remote ]]
+      volumes_results_remote: $[[ volumes.results.remote ]]
+```
+
+## Arguments
+
+### `image`
+
+The name of the image on which to run the VS Code instance.
```

Comment:
I would add something along the lines: "Default is `neuromation/base`. If you use an image not derived from `neuromation/base` make sure it has the vscode server (https://github.com/cdr/code-server) installed"

Reply:
Fixed in the latest commit